### PR TITLE
Trim codeowners

### PR DESCRIPTION
**Pull request by brokensound77**
## Summary

Trim codeowners

### CODE REVIEW

File: .github/CODEOWNERS
Comment (Mikaayenson) [inline .github/CODEOWNERS:8]:
```suggestion
tests/**/*.py     @mikaayenson @eric-forte-elastic @terrancedejesus 
detection_rules/  @mikaayenson @eric-forte-elastic @terrancedejesus 
tests/            @mikaayenson @eric-forte-elastic @terrancedejesus 
lib/              @mikaayenson @eric-forte-elastic @terrancedejesus 
rta/              @mikaayenson @eric-forte-elastic @terrancedejesus 
```

File: .github/CODEOWNERS
Comment (Mikaayenson) [inline .github/CODEOWNERS:14]:
```suggestion
detection_rules/etc/packages.yaml @mikaayenson @eric-forte-elastic @terrancedejesus 
detection_rules/etc/*.json        @mikaayenson @eric-forte-elastic @terrancedejesus 
detection_rules/etc/*.json        @mikaayenson @eric-forte-elastic @terrancedejesus 
detection_rules/etc/*/*           @mikaayenson @eric-forte-elastic @terrancedejesus 
```

**Review by terrancedejesus — APPROVED** [review]
Looks good.

**Review by shashank-elastic — APPROVED** [review]
Approving provided we have Terrance added as per suggestion to codeowners

**Pull request merged**
